staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -739,7 +739,7 @@ type EmptyDirVolumeSource struct {
 	// The default is nil which means that the limit is undefined.
 	// More info: http://kubernetes.io/docs/user-guide/volumes#emptydir
 	// +optional
-	SizeLimit resource.Quantity `json:"sizeLimit,omitempty" protobuf:"bytes,2,opt,name=sizeLimit"`
+	SizeLimit *resource.Quantity `json:"sizeLimit,omitempty" protobuf:"bytes,2,opt,name=sizeLimit"`
```

Comment:
Doesn't this break backward compatibility?

Reply:
Actually, the introduction of the non-pointer field broke compatibility, and this restored it